src/models/ServerAcl.ts

```diff
@@ -104,6 +104,9 @@ export class ServerAcl {
         if (!allowed || allowed.length === 0) {
```

Comment:
Wait a minute, `!allowed`?? What's that supposed to mean? And how are we pushing to it if its undefined??

Reply:
Probably should make this `allowed = [ '*' ];`

Reply:
Why?

As for why `!allowed` was there, seemingly it was a redundant check from a long time ago